api-js/src/SDKMeta.ts

```diff
@@ -29,9 +30,41 @@ export const Languages: Record<string, string[]> = sdkLanguages;
 export const Names: Record<string, string> = sdkNames;
 export const Repos: Record<string, Repo> = sdkRepos;
 export const Popularity: Record<string, number> = sdkPopularity;
+export const Releases: ReleaseList = Object.fromEntries(
+    Object.entries(sdkReleases).map(([key, value]) => [
+      key,
+      value.map((release: any) => ({
+          Major: release["major"],
+          Minor: release["minor"],
+          Date: new Date(release["date"]),
+          EOL: release["eol"] ? new Date(release["eol"]) : null
+      }))
+    ]));
 
 export const Types: Record<string, Type> = Object.fromEntries(
     Object.entries(sdkTypes).map(([key, value]) => [
       key,
       isType(value) ? value : Type.Unknown
     ]));
+
+
+export interface Release {
+    Major: number;
+    Minor: number;
+    Date: Date;
+    EOL: Date | null;
+}
+
+export interface ReleaseList {
+    [key: string]: Release[];
+}
+
+export namespace ReleaseHelpers {
```

Comment:
Might not be the most idiomatic thing (namespace)?

I wanted to _not_ suffix each of these functions with `Release` or something similar.